gpustack/migrations/versions/2025_09_08_1454-924c9a0b4c13_v2_0_database_migration.py

```diff
@@ -88,12 +88,15 @@ def upgrade() -> None:
         sa.Column('deleted_at', sa.DateTime(), nullable=True),
     )
     op.create_index('idx_clusters_deleted_at_created_at', 'clusters', ['deleted_at', 'created_at'])
-    # create default cluster
-    # stat == 3 means PROVISIONED and READY
-    op.execute(f"""
-        INSERT INTO clusters (name, description, provider, state, hashed_suffix, registration_token, created_at, updated_at)
-        VALUES ('Default Cluster', 'The default cluster for GPUStack', 'Docker', 'READY', '{secrets.token_hex(6)}', '', {utc_now_sql_literal()}, {utc_now_sql_literal()})
-    """)
+
+    gpustack_config = gpustack.config.config.get_global_config()
```

Comment:
Please add default value for `enable_worker` to avoid `gpustack_config` is None. e.g.
```python
gpustack_config = gpustack.config.config.get_global_config()
enable_worker = gpustack_config.enable_worker if gpustack_config else False.
```